DT-364 Migrate Epiphany to Core22/Gnome42

sergio-costas opened the pull request.

sergio-costas committed:
DT-364 Migrate Epiphany to Core22/Gnome42

sergio-costas committed:
Remove double launcher

sergio-costas committed:
Updated to last version

jssotomdz commented on snapcraft.yaml:
```diff
@@ -162,12 +113,11 @@ parts:
       - libglu1-mesa
       - libgpm2
-      - libhogweed5
-      - libicu66
```
some of these packages are already provided by the gnome sdk https://github.com/ubuntu/gnome-sdk/blob/gnome-42-2204-sdk/snapcraft.yaml

sergio-costas replied:
Done. Also added metainfo file and removed "craftctl set version". Thanks!

sergio-costas committed:
Added metainfo and removed unneeded packages

'craftctl set version' is not needed when a parse-info statement
is available.

Also removed several unneeded packages.

hellsworth commented:
looks good thanks!
